packages/xo-server/src/xo-mixins/remotes.mjs

```diff
@@ -43,6 +43,12 @@ function validateUrl(url) {
       warn(new Error('invalid remote URL param ' + param))
     }
   }
+
+  const hasEncryption = encryptionKey !== undefined
+  const hasVhdDirectory = useVhdDirectory === true
+  if (hasEncryption && !hasVhdDirectory) {
+    throw invalidParameters('Encryption must not be used without using VHD directory (data blocks)')
```

Comment:
I found this hard to understand with the double negative "**do not** use **without**".
What about: `Encryption must be used with VHD directory (data blocks)`